src/Dapr.Actors/Runtime/ConverterUtils.cs

```diff
@@ -60,5 +63,69 @@ public static string ConvertTimeSpanValueInDaprFormat(TimeSpan? value)
 
             return stringValue;
         }
+
+        public static string ConvertTimeSpanValueInISO8601Format(TimeSpan value, int repetitions)
+        {
+            if (repetitions == -1)
+            {
+                return ConvertTimeSpanValueInDaprFormat(value);
+            }
```

Comment:
Why the separate path? Why not just use the repetition as normal?

Reply:
There may be reminder periods still created in golang's time.Duration format. When we added the repetition to runtime, we wanted to support both time.Duration format and ISO 8601 format. hence the separate paths.